Xamarin/SSW.Consulting/SSW.Consulting.Android/Properties/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="7" android:versionName="1.5" package="com.ssw.consulting" android:installLocation="auto">
-	<uses-sdk android:minSdkVersion="24" android:targetSdkVersion="28" />
+	<uses-sdk android:minSdkVersion="24" android:targetSdkVersion="27" />
```

Comment:
Why are we lowering the target sdk version?

Reply:
reverted